awx/api/serializers.py

```diff
@@ -1756,6 +1745,14 @@ def to_representation(self, obj):
             ret['last_job_host_summary'] = None
         return ret
 
+    def get_has_active_failures(self, obj):
+        return bool(
+            obj.last_job_host_summary and obj.last_job_host_summary.failed
```

Comment:
Oh, so I guess we're already fetching this information and showing it in summary_fields. And fetching it... and even more

https://github.com/ansible/awx/blob/90d38a50de42f403733991eba4e3ffbf4ca61678/awx/main/access.py#L908-L909

Reply:
![](https://media.giphy.com/media/XMBJ0l20sNWEM/giphy.gif)